app/api/i18n/translations.js

```diff
@@ -129,14 +129,34 @@ export default {
               return Promise.resolve();
             }
             context.values = context.values || [];
-            context.values.push({ key, value: defaultValue });
+            const value = context.values.find(val => val.key === key);
+            if (value) {
+              const index = context.values.indexOf(value);
+              context.values[index].value = defaultValue;
```

Comment:
this condition is not tested.

Reply:
Actually that change is not needed, I will revert to how it was previously

Reply:
reverted